docs/metrics.md

```diff
@@ -114,6 +114,9 @@ Total VM filesystem capacity in bytes. Type: Gauge.
 ### kubevirt_vmi_filesystem_used_bytes
 Used VM filesystem capacity in bytes. Type: Gauge.
 
+### kubevirt_vmi_last_api_connection_timestamp_seconds
+Virtual Machine Instance last API connection timestamp. Including VNC, console, portforward, SSH and usbredir connections. Type: Gauge.
```

Comment:
@sradco Following a discussion I had with @0xFelix, I've updated the metric name and description, to mention the connection is made via the API (which both virtctl and UI are using).
Using `_connection_` alone might be misleading, since the user can create a service to expose the VM and than ssh/vnc to it (without using virtctl), and then the metric won't tell us about it. Seems like there isn't a proper way to cover it in kubevirt level, so better to be more specific in the metric name and description.

Reply:
Once we agree on the name for the new metric and merge this PR, I'll open a bug to align the other connection metrics (e.g. [kubevirt_vnc_active_connections](https://github.com/kubevirt/kubevirt/blob/main/docs/metrics.md#kubevirt_vnc_active_connections)) with the more accurate naming, as agreed with @0xFelix.